utils/build-tooling-libs

```diff
@@ -499,6 +501,16 @@ Example invocations:
         default=default_ninja,
         help="the path to ninja (default = %s)" % default_ninja,
     )
+    option(
+        "--swiftc",
```

Comment:
Not a strong opinion but `--swiftc-path` would be more consistent with `--ninja-path`

Reply:
You’re right 👍 Thanks.